homeassistant/components/telegram_bot/__init__.py

```diff
@@ -75,17 +77,34 @@
 PARSER_HTML = 'html'
 PARSER_MD = 'markdown'
 
-PLATFORM_SCHEMA = cv.PLATFORM_SCHEMA.extend({
-    vol.Required(CONF_PLATFORM): vol.In(('broadcast', 'polling', 'webhooks')),
-    vol.Required(CONF_API_KEY): cv.string,
-    vol.Required(CONF_ALLOWED_CHAT_IDS):
-        vol.All(cv.ensure_list, [vol.Coerce(int)]),
-    vol.Optional(ATTR_PARSER, default=PARSER_MD): cv.string,
-    vol.Optional(CONF_PROXY_URL): cv.string,
-    vol.Optional(CONF_PROXY_PARAMS): dict,
-})
-
-PLATFORM_SCHEMA_BASE = cv.PLATFORM_SCHEMA_BASE.extend(PLATFORM_SCHEMA.schema)
+DEFAULT_TRUSTED_NETWORKS = [
+    ip_network('149.154.167.197/32'),
+    ip_network('149.154.167.198/31'),
+    ip_network('149.154.167.200/29'),
+    ip_network('149.154.167.208/28'),
+    ip_network('149.154.167.224/29'),
+    ip_network('149.154.167.232/31')
+]
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.All(cv.ensure_list, [
+        vol.Schema({
+            vol.Required(CONF_PLATFORM): vol.In(
+                ('broadcast', 'polling', 'webhooks')),
+            vol.Required(CONF_API_KEY): cv.string,
+            vol.Required(CONF_ALLOWED_CHAT_IDS):
+                vol.All(cv.ensure_list, [vol.Coerce(int)]),
+            vol.Optional(ATTR_PARSER, default=PARSER_MD): cv.string,
+            vol.Optional(CONF_PROXY_URL): cv.string,
+            vol.Optional(CONF_PROXY_PARAMS): dict,
+            # webhooks
+            vol.Optional(CONF_URL): vol.Url(),
```

Comment:
OK... this is literally https://github.com/home-assistant/home-assistant/pull/23022/files#diff-da552d1dd826a1d7c9202937091c322fL84 yet before there was no lint error and now there is :(